packages/compass-aggregations/src/components/settings/settings.jsx

```diff
@@ -76,14 +76,16 @@ class Settings extends PureComponent {
       return (
         <div className={classnames(styles['input-group'])}>
           <div className={classnames(styles['input-meta'])}>
-            <label>Limit</label>
-            <p>
+            <label htmlFor='aggregation-limit'>Limit</label>
```

Comment:
Some accessibility issues caught by eslint that were easy enough to address right away